as2_python_api/as2_python_api/behavior_actions/followpath_behavior.py

```diff
@@ -118,7 +120,7 @@ def __get_path(self, path: Union[list, tuple, Path, GeoPath, PoseWithID]):
                 # CAUTION: using height from origin
                 point_list.append([x, y, pose.pose.position.altitude])
         elif isinstance(path, PoseWithID):
-            return list(path)
+            return list([path])
```

Comment:
can this be replaced with `return [path]` ?

Reply:
That's what flake8 is complaining about:
```
  as2_python_api/behavior_actions/traj_generation_behavior.py:111:20: C410 Unnecessary list passed to list() - remove the outer call to list().
              return list([path])
```